lib/index.js

```diff
@@ -71,14 +69,15 @@ function EslintValidationFilter(inputTree, options) {
   }
   this.options = options || {};
   this.eslintOptions = options.options || {};
-  // set inputTree
-  this.inputTree = inputTree;
+  // call base class constructor
+  Filter.call(this, inputTree);
 
   // set formatter
-  this.formatter = require(this.options.format || 'eslint/lib/formatters/stylish');
+  this.formatter = require(this.options.format || 'eslint/lib/formatters/stylish'); // eslint-disable-line global-require
 
   this.cli = new CLIEngine(this.eslintOptions);
 
+  this.eslintrc = inputTree;
```

Comment:
Good point. All the tests provide a string as the input node, so I overlooked the possibility that this could be a tree instance. Any ideas how to resolve this from a tree? My first thought is to recursively resolve the `_inputNodes` objects until we get a string, but this doesn't account for merge trees.